pkg/compose/compose.go

```diff
@@ -45,7 +45,8 @@ func NewComposeService(dockerCli command.Cli) api.Service {
 }
 
 type composeService struct {
-	dockerCli command.Cli
+	dockerCli      command.Cli
+	maxConcurrency int
```

Comment:
shouldn't it be initialize with a `-1` value by default as the default value for an int will be `0`